gtsfm/densify/mvs_patchmatchnet.py

```diff
@@ -0,0 +1,251 @@
+"""Inference methods for GtsfmData in PatchmatchNet
+
+Authors: Ren Liu
+"""
+import copy
+import time
+from typing import Dict, List
+
+import numpy as np
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader
+
+from gtsfm.common.image import Image
+from gtsfm.common.gtsfm_data import GtsfmData
+from gtsfm.densify.mvs_base import MVSBase
+from gtsfm.densify.patchmatchnet_data import PatchmatchNetData
+from gtsfm.utils import logger as logger_utils
+from thirdparty.patchmatchnet.models.net import PatchmatchNet
+from thirdparty.patchmatchnet.utils import tensor2numpy, tocuda
+from thirdparty.patchmatchnet.eval import check_geometric_consistency
+
+logger = logger_utils.get_logger()
+
+torch.backends.cudnn.benchmark = True
+PATCHMATCHNET_WEIGHTS_PATH = "thirdparty/patchmatchnet/checkpoints/model_000007.ckpt"
+DEFAULT_VIEW_NUMBER = 5
+DEFAULT_GEOMETRIC_PIXEL_THRESH = 1.0
```

Comment:
maybe we can add an inline comment explaining where these threshold values come from? paper / link / url?

Reply:
thanks, marking as resolved